includes/ssp-functions.php

```diff
@@ -924,4 +924,19 @@ function ssp_setup_upload_credentials() {
 		return compact( 'bucket', 'show_slug', 'episodes_url', 'access_key_id', 'policy', 'signature' );
 		
 	}
+}
+
+/**
+ * Get image ID when only the URL of the image is known
+ */
+if( !function_exists( 'ssp_get_image_id_from_url' ) ){
+    /**
+     * @param $image_url
+     * @return mixed
+     */
+    function ssp_get_image_id_from_url( $image_url ){
+        global $wpdb;
+        $attachment = $wpdb->get_col( $wpdb->prepare("SELECT ID FROM $wpdb->posts WHERE guid='%s';", $image_url ) );
+        return $attachment[0];
```

Comment:
Once HTML5 player enabled, getting the following error 
Notice: Undefined offset: 0 in /var/www/public/wp-content/plugins/seriously-simple-podcasting/includes/ssp-functions.php on line 940
Probably because the episode doesn't have a featured image.

Reply:
Thanks for reporting this, I have added a control mechanism:

`return isset( $attachment[0] ) ? $attachment[0] : false;`

Kindly confirm if this error has disappeared for you.